client/components/tinymce/plugins/wpcom/wpcom-utils.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Removes empty spaces from empty paragraphs
+ * This logic is borrowed from core's TinyMCE Plugin
+ *
+ * @see https://core.trac.wordpress.org/browser/trunk/src/wp-includes/js/tinymce/plugins/wordpress/plugin.js#L133
```

Comment:
nice! although this reference will lose sync because it links to `trunk`. it might be useful to link to the commit that introduced this fix - even more context and an _always-valid_ link